articles/python/azure-sdk-sovereign-domain.md

```diff
@@ -67,11 +71,13 @@ stack_cloud = get_cloud_from_metadata_endpoint("https://contoso-azurestack-arm-e
 # https:// is optional in the URL but required on the endpoint.
 credential = DefaultAzureCredential(authority=stack_cloud.endpoints.active_directory)
 
-resource_client = ResourceManagementClient(credential, subscription_id,
+resource_client = ResourceManagementClient(
+    credential, subscription_id,
     base_url=stack_cloud.endpoints.resource_manager,
-    credential_scopes=[cloud.endpoints.resource_manager + ".default'"])
+    credential_scopes=[stack_cloud.endpoints.active_directory_resource_id + "/.default"])
```

Comment:
`resource_manager` may differ from `active_directory_resource_id` and is not valid `resource` in Azure Stack.

For example, for **redmond** Azure Stack:

- `resource_manager`: `https://management.redmond.azurestack.corp.microsoft.com/`
- `active_directory_resource_id`: `https://management.adfs.redmond.selfhost.local/81cfafcd-8b9a-4419-9c85-823b54076b55`